### ISSUE TITLE
Feature request to replay fzf command + arguments

### ISSUE THREAD

**Issue by pinkchry** [junegunn/fzf.vim#618]
- Category
    - [ ] Question
    - [ ] Bug
    - [ x] Suggestion
- OS
    - [ ] Linux
    - [ ] macOS
    - [ ] Windows
    - [ ] Etc.
- Vim
    - [ ] Vim
    - [ ] Neovim

It would be nice to have a way to replay the last fzf command (including the arguments that were typed in on the prompt).

**Comment by Piping** [comment]
type `@:` repeat last command in vim.

**Comment by ggsalas** [comment]
@Piping `@:` works if you use a command like `:Ag string`.
But if you do `:Ag` and then search `string` you can't come back to your last search, you need to type again `string`.

Would be nice to have a way to come back to the last search.

**Comment by Piping** [comment]
@ggsalas yeah, in general if FZF.vim supports some history that will be awesome.

**Comment by bdlangton** [comment]
I also think this would be a great feature to add. I'm not great with vimscript, but I'll try to look at ways to accomplish this.
